build.sbt

```diff
@@ -2,7 +2,7 @@ Global / onChangedBuildSource := ReloadOnSourceChanges
 
 inThisBuild(
   Seq(
-    organization := "com.github.lolgab",
+    organization := "dev.whaling",
```

Comment:
Since I'm maintaining the library @rwhaling proposed to use my groupid instead of `"dev.whaling"`
You should revert this change.

Reply:
:+1:

Reply:
Btw in future we should probably migrate to `org.scala-native`, this repo is already hosted at `github.com/scala-native` anyway.